src/styles/foundation/_focus-ring.scss

```diff
@@ -21,15 +20,15 @@
       content: var(--p-non-null-content, none);
       position: absolute;
       z-index: 1;
-      top: $negative-offset;
-      right: $negative-offset;
-      bottom: $negative-offset;
-      left: $negative-offset;
+      top: $offset;
+      right: $offset;
+      bottom: $offset;
+      left: $offset;
       display: block;
       pointer-events: none;
-      box-shadow: 0 0 0 $negative-offset var(--p-focused);
+      box-shadow: 0 0 0 $offset var(--p-focused);
       transition: box-shadow duration(fast) var(--p-ease);
-      border-radius: calc(#{$border-radius} + #{rem(1px)});
+      border-radius: calc(#{$border-radius} + #{$border-width} + #{rem(1px)});
```

Comment:
The border radius was off on some elements. I've tested this with the kitchen sink and it seems to match every element perfectly